testing/gearmand/APKBUILD

```diff
@@ -0,0 +1,75 @@
+# Contributor: Aaron Hurt <ahurt@ena.com>
+# Maintainer: Aaron Hurt <ahurt@ena.com>
+pkgname=gearmand
+pkgver="1.1.12"
+pkgrel=2
+pkgdesc="Gearman job server and client"
+url="http://gearman.org"
+arch="all"
+license="BSD"
+depends="libgearman"
+depends_dev="boost-dev gperf libevent-dev util-linux-dev mariadb-dev
+	hiredis-dev openssl-dev sqlite-dev libmemcached-dev"
+makedepends="$depends_dev"
+install="$pkgname.pre-install $pkgname.post-deinstall"
+subpackages="$pkgname-doc libgearman libgearman-dev"
+source="https://launchpad.net/${pkgname}/1.2/${pkgver}/+download/${pkgname}-${pkgver}.tar.gz
+	libtest-cmdline.cc.patch
+	$pkgname.initd
+	$pkgname.confd"
+
+builddir="$srcdir/$pkgname-$pkgver"
+build() {
+	cd "$builddir"
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--localstatedir=/var \
+		--disable-libpq \
```

Comment:
We don't use persistence in our deployment and it's usually not for short running jobs.  Persistence really only makes sense for long running jobs and then you might be better off using RabbitMQ or something similar.